n8n/helm/n8n/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   repository: dkr.plural.sh/n8n/n8nio/n8n
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "0.202.1"
+  tag: "0.212.1"
```

Comment:
It looks like the latest version is `0.213.0` based on the image we have on our repository.

Reply:
done !